src/lightgbm/src/main/python/LightGBMClassifier.py

```diff
@@ -26,13 +27,40 @@ def saveNativeModel(self, sparkSession, filename):
         """
         Save the booster as string format to a local or WASB remote location.
         """
-        ctx = SparkContext.getOrCreate()
-        sql_ctx = SQLContext.getOrCreate(ctx)
-        jsession = sql_ctx.sparkSession._jsparkSession
+        jsession = sparkSession._jsparkSession

```

Comment:
likewise there is a python equivalent to SparkSession.getOrCreate. Its something like getActiveSession

Reply:
done!